mev_inspect/aave_liquidations.py

```diff
@@ -0,0 +1,65 @@
+from typing import List
+
+from mev_inspect.schemas.classified_traces import (
+    ClassifiedTrace,
+    DecodedCallTrace,
+    Classification,
+)
+
+from mev_inspect.schemas.liquidations import Liquidation
+
+
+LIQUIDATION_CONTRACT_ADDRESSES = [
+    "0x3dfd23A6c5E8BbcFc9581d2E864a68feb6a076d3",
+    "0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9",
+    "0x398eC7346DcD622eDc5ae82352F02bE94C62d119",
+]
+
+
+def find_liquidations_from_traces(
+    traces: List[ClassifiedTrace],
+) -> List[Liquidation]:
+    """Inspect list of classified traces and identify liquidation"""
+    seen_transactions: List[str] = []
+    liquidations: List[ClassifiedTrace] = []
+    result: List[Liquidation] = []
+
+    for trace in traces:
+
+        if isinstance(trace, DecodedCallTrace):
+
+            # Check for liquidation and register trace and unique liquidator
+            if (
+                trace.classification == Classification.liquidate
+                and trace.transaction_hash not in seen_transactions
+            ):
+
+                liquidations.append(trace)
+                LIQUIDATION_CONTRACT_ADDRESSES.append(trace.from_address)
```

Comment:
woah wait so it's not constant?

(shows the value of differentiating with caps for constants - things like this stand out visually)